bip-0131.mediawiki

```diff
@@ -77,6 +77,26 @@ the user when their wallet contains many UTXOs that qualify it to benefit from
 a coalescing transaction. Wallets should not simply replace non-coalescing transactions
 with coalescing transactions in all instances.
 
+== Isn't this BIP bad because it encourage address re-use? ==
+
+Address re-use comes in two forms: re-use by the ''sender'', and re-use by the ''receiver''.
+
+Re-use by the sender is basically using the same address for the change output. This is generally considered bad
+since people looking through your transaction history can determine who you do business with. When
+you generate a new address for every change, your privacy is conserved as it is impossible to know which
+output is a recipient, and which output is the change output. This BIP has '''no effect''' on re-use
+by the sender.
+
+On the other hand, address re-use by the ''receiver'' occurs under completely different circumstances.
+When you publish an address and have multiple people send to that address, you are engaging in address re-use
+from the receiver. This activity has historically been considered bad because it leads to re-using a private key.
+When you re-use a private key too many times, you run the risk of an attacker performing statistical analysis
+on the multiple signatures, which can lead to an attacker finding out your private key.
```

Comment:
If somebody wants to give up some privacy for a massive decrease in transaction fees, then they should be able to do that. There are many many addresses that have multiple unspent outputs associated with them on the network today, proving that the decrease in privacy is acceptable to many people.

Technically, when you are going through the checkout at Walmart, you can look in to the cash register and get as much information as you would if you looked through someone's bitcoin history through a blockexplorer. Neither Walmart nor any other retailer finds it necessary to keep a privacy guard over the cash register. As long as you rotate your address every so often, its going to take a lot of work to figure out someone's complete financial story by through a block explorer.

By the way, if **I** re-use my address, that has no effect on anyone but myself. Other people are not affected at all.